src/main/java/plugily/projects/villagedefense/arena/states/InGameState.java

```diff
@@ -82,9 +85,29 @@ public void handleCall(PluginArena arena) {
       if(arena.getArenaOption("ZOMBIES_TO_SPAWN") < 0) {
         arena.setArenaOption("ZOMBIES_TO_SPAWN", 0);
       }
-    } else if(arena.getTimer() <= 0) {
-      pluginArena.setFighting(true);
-      pluginArena.getPlugin().getArenaManager().startWave(pluginArena);
+    } else if(arena.getTimer() <= 3) {
+      playStartSound((Arena) arena);
+      if(arena.getTimer() <= 0) {
+        pluginArena.setFighting(true);
+        pluginArena.getPlugin().getArenaManager().startWave(pluginArena);
+      } else if(arena.getTimer() >= 1) {
+        new TitleBuilder("IN_GAME_MESSAGES_VILLAGE_WAVE_TITLE_STARTING_" + arena.getTimer()).asKey().arena(arena).sendArena();
+      }
+    }
+  }
+
+  private void playStartSound(Arena arena) {
+    for(Player player : arena.getPlayers()) {
+      switch(arena.getTimer()) {
+        case 3:
+        case 2:
+        case 1:
+          XSound.BLOCK_NOTE_BLOCK_HAT.play(player);
+          break;
+        case 0:
+          XSound.BLOCK_NOTE_BLOCK_FLUTE.play(player);
+          break;
+      }
```

Comment:
I am adding this to MinigamesCore Starting Stage, too, so if game begins the last seconds will get this new sound before the game starts ;) Leave it like it is. Just a comment